### PR TITLE
fix some capitalization in script commands and functions

### PR DESCRIPTION
**Pull request by jokoho48**
if someone found some more capitalization issues post it below i can than take care of it

### CODE REVIEW

**Comment by jonpas** [comment]
:+1:

**Comment by commy2** [comment]
:+1: from me

**Comment by jokoho48** [comment]
i am not done wit that

**Comment by PabstMirror** [comment]
:+1: 

This may cause some conflicts with other PRs, but they should be trivial to fix

**Comment by commy2** [comment]
This is gonna be fun task to kill time on my week end.

File: addons/zeus/functions/fnc_bi_moduleRemoteControl.sqf [hunk 1]
```diff
@@ -117,32 +117,32 @@ if (_activated && local _logic && !isnull curatorcamera) then {
...
             player addrating (-rating player + _rating);
-            objnull remotecontrol _unit;
-            _unit setvariable ["bis_fnc_moduleRemoteControl_owner",nil,true];
+            objNull remotecontrol _unit;
+            _unit setVariable ["bis_fnc_moduleRemoteControl_owner",nil,true];
 
             //--- Death screen
             if (
-                isnull curatorcamera
+                isNill curatorcamera
```
Comment (commy2) [inline addons/zeus/functions/fnc_bi_moduleRemoteControl.sqf]: : )

**Pull request closed without merging**